### DISCUSSION TITLE
2024-08-01 목요일

### DISCUSSION THREAD

**Discussion by eunohhh** [nbc-final-a10/TripBuddy#169] [기록보드]
2024-08-01 목요일 기록보드입니다.

## 여행가고십조(A10) 일일 공통 일과표

| 시간 | 일과 |  기록 |
| :-: | :-: | :-: |
| 09:00 ~ 10:00 | 아침 회의 | |
| 10:00 ~ 11:00 | 기술면접 준비 | |
| 11:00 ~ 12:00 | 집중 개발 time | |
| 12:00 ~ 13:00 |집중 개발 time |  |
| 13:00 ~ 14:00 | 점심 식사 | |
| 14:00 ~ 18:00 | 집중 개발 time | |
| 18:00 ~ 19:00 | 저녁 식사 | |
| 19:00 ~ 20:00 | 저녁 회의 | |
| 20:00 ~ 21:00 | 5분 기록보드 작성 / 개인 회고 | |

```
## 개인 기록보드 서식 (김병준)

### 오늘 한 일

- 내용

### 트러블 슈팅

- 내용

### 내일 계획

- 내용
```

```
// 이미지 테이블
<table>
  <tr>
    <th>이미지 1번 제목</th>
    <th>이미지 2번 제목</th>
  </tr>
  <tr>
    <td><img src="이미지 주소" width="300" height="300"/></td>
    <td><img src="이미지 주소" width="300" height="300"/></td>
  </tr>
</table>
```

**Comment by eunohhh** [comment]
## 개인 기록보드 서식 (오은)

### 오늘 한 일

- 스토리 관련 기능 전반적으로 작성
- 전반적인 레이아웃 손보기
- 메인페이지 스토리, 여정 카드 리스트 렌더링 작성

### 트러블 슈팅

- 서버에서 sharp 라이브러리를 쓸 경우, 최신버전은 vercel 배포 오류납니다. 0.32.6 버전으로 다운그레이드하여 해결했습니다.

### 내일 계획

#### 전체적인 퍼블리싱 체크
- 장소선택 탭 전체조율 할 것(스크롤 만들기, 고정높이 정하기)
- 메인페이지 조율할 것(특히  여행자님 대만이... 이부분하고 배경사진 / 검색바 어떻게 할지)
- 페이지간 연결성 확인(어디 누르면 어디로 가고 이런거)
- 탭메뉴 좌우폭(모바일 좌우폭에 맞추기)
- 모달 디자인 확인(얼러트 모달 떳을때 z-index 낮아서 헤더, 하단바가 더 위에 보이는 증상)
- 로그아웃 버튼 위치 <=== 마이페이지로
- 로더 디자인 부탁할것

#### 로직관련
- 부가적인 테이블들 진행상황(좋아요, 팔로우 등)체크(리마인드만)
- 특히 contract 테이블

**Comment by Captain-Kim** [comment]
## 완료 혹은 진행 중인 Task
- 게시글 작성 기능
- 홈 화면 (추천인기버디즈 목록 렌더링)

## 오늘 진행 예정 혹은 완료 예정인 Task
- 홈 화면 (추천버디즈 목록) 세부 스타일링
- 마이페이지 회원정보 수정기능 구현
- 게시글 유효성 검사

**Comment by slsl2** [comment]
## 개인 기록보드 서식 (김민지)

### 오늘 한 일

- 채팅 목록 페이지 데이터 연결

### 내일 계획

- home 페이지 배너 데이터 연결 및 이미지 처리
- 채팅 목록 & 채팅 페이지 데이터 연결 완성
